src/validators/ObjectValidator.ts

```diff
@@ -80,7 +80,7 @@ export class ObjectValidator<T extends NonNullObject, I = UndefinedToOptional<T>
 		}
 	}
 
-	public get strict(): ObjectValidator<{ [Key in keyof I]-?: I[Key] }> {
+	public get strict() {
```

Comment:
Is this correct?

Reply:
yes. Strict doesn't changes the type it just makes sure no unknown keys are passed.